streaq/ui/deps.py

```diff
@@ -15,3 +16,14 @@ def get_worker() -> Worker[Any]:
         status_code=status.HTTP_412_PRECONDITION_FAILED,
         detail="get_worker dependency not implemented!",
     )
+
+
+def get_result_formatter() -> Callable[[Any], str]:
+    return str
```

Comment:
Can you have this retain the truncation behavior as a default?

Reply:
Yes, we can use `textwrap.shorten` function to truncate the result to fit in the given width.

```
shorten(text, width, **kwargs)
    Collapse and truncate the given text to fit in the given width.

    The text first has its whitespace collapsed.  If it then fits in
    the *width*, it is returned as is.  Otherwise, as many words
    as possible are joined and then the placeholder is appended::

textwrap.shorten("Hello  world!", width=12)
        'Hello world!'
textwrap.shorten("Hello  world!", width=11)
        'Hello [...]'
```

Reply:
Perfect!

Reply:
I actually misread this, for some reason I confused the truncation here with the truncation done in the logs. I'm going to revert this, sorry for making you do extra work!